src/amo/sagas/home.js

```diff
@@ -60,121 +53,57 @@ export function* fetchHomeData({
     },
   };
 
+  const recommendedExtensionShelf = (
+    addons: Array<ExternalAddonType>,
+  ): ExternalResultShelfType => {
+    return {
+      title: selectLocalizedContent('Recommended Extensions', state.lang),
```

Comment:
What you are creating here is meant to be an external representation of the data (i.e., what we would get back from an API), so you don't want to use `selectLocalizedContent` here, as that's used for creating an internal representation. I think what we actually want here is `createLocalizedString`. The problem with that is that it lives in `tests/unit/helpers.js` and we can't really use that in our code, so it will need to be moved.

I think we can make that a follow-up, so for now you should just contruct a LocalizedString object in a way similar to what `createLocalizedString` does here. Because you'll need to do it twice (once here, and once for `footer.text`, you should create a local helper function for it.